index.html

```diff
@@ -2595,6 +2595,139 @@ <h3>Data Schemas</h3>
+        </p>
+        <p>
+          To validate that a resource referenced by a Verifiable Credential is
+          the same at verification time as at issuing time, an implementer
+          MAY include a property named <code>resourceIntegrity</code> that
+          stores an array of objects that describe additional integrity
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the [[URL]] to the resource named <code>id</code> and the 
+          <code>integrity</code> information for the resource
+          constructed using the method specified in <a
+          href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource
+          Integrity</a>.
+        </p>
+        <p>
+          There MUST NOT be more than one object in the
+          <code>resourceIntegrity</code> per <code>id</code>. 
+        </p>
+        <p>
+          When performing integrity checking the
+          strongest hash algorithm available SHOULD be used to confirm
+          integrity of the resource.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY
+          contain a property named <code>timestamp</code>
+          that indicates the time at which the hash was computed. The
+          <code>timestamp</code> property if included MUST be a string
+          value of an [[XMLSCHEMA11-2]] combined date-time string.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          a property named <code>mediaType</code> that indicates the 
+          expected media type for the indicated <code>resource</code>.
+          If a <code>mediaType</code> is included it must be a valid
+          media type as listed in the 
+          <a href="https://www.iana.org/assignments/media-types/media-types.xhtml">
+            IANA Media Types
+          </a> registry.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          additional properties.
+        </p>
+        <p>
+          In any object in the <code>credential.credentialSubject</code>
+          that contains a [[URL]], a property named
+          <code>integrity</code> may be included with the integrity
+          information as specified above, with the [[URL]] in the
+          <code>id</code> property. 
+        </p>
+        <p>
+          Any objects for which selective disclosure is desired SHOULD NOT
+          be included as an object in the <code>resourceIntegrity</code>
+          array.
+        </p>
+        <p>
+          Implementers SHOULD consult appropriate sources, such as the 
```

Comment:
```suggestion
          Implementers are urged to consult appropriate sources, such as the 
```

SHOULD here is not testable.